app/pi-prebuild.sh

```diff
@@ -17,9 +17,9 @@ done
 
 if [ "$no_imgui" == true ]
 then
-    VCPKG_BRANCH="2022.01.01" VCPKG_FORCE_SYSTEM_BINARIES=1 "${SCRIPT_DIR}"/linux-prebuild.sh -n
+    VCPKG_BRANCH="2022.02.02" VCPKG_FORCE_SYSTEM_BINARIES=1 "${SCRIPT_DIR}"/linux-prebuild.sh -n
```

Comment:
Could we change this in linux-prebuild.sh too? I'm also not sure the vcpkg branch needs to get defined here at all if the value is the same as in linux-prebuild.sh, unless they are expected to diverge for whatever reason (and I guess it could be added back easily when it does diverge if so)

Reply:
ooh, I thought I had done that - this is just an error on my behalf. Definitely think we shoudl try and keep the vcpkg stuff as up-to-date and inline across platforms as possible. Also, the versions have diverged in the past between generic linux and pi-builds which is why they're currently separate. Happy to see them merged again.

Reply:
I think merging and only using the default in linux-prebuild.sh for now is appropriate until/if it's a problem again (less places to change for updates). But I also don't feel that strongly either way, as long as default in linux-prebuild.sh gets updated too